InstantPrintTool.py

```diff
@@ -41,11 +46,14 @@ def __init__(self, iface, populateCompositionFz=None):
         self.oldrubberband = None
         self.pressPos = None
         self.printer = QPrinter()
+        self.mapitem = None
         self.populateCompositionFz = populateCompositionFz
 
         self.dialog = InstantPrintDialog(self.iface.mainWindow())
         self.dialogui = Ui_InstantPrintDialog()
         self.dialogui.setupUi(self.dialog)
+        self.dialogui.addScale.setIcon(QIcon(":/plugins/instantprint/icons/mActionAdd.svg"))
```

Comment:
":/images/themes/default/mActionAdd.svg", no need to separately ship an icon which is already shipped by qgis.